river/Cursor.zig

```diff
@@ -847,8 +849,6 @@ fn surfaceAtFilter(view: *View, filter_tags: u32) bool {
 pub fn enterMode(self: *Self, mode: enum { move, resize }, view: *View) void {
     log.debug("enter {s} cursor mode", .{@tagName(mode)});
 
-    self.seat.focus(view);
```

Comment:
Hmm... I can not come up with an example right now, but I have the feeling that this might cause some issues somewhere. Button mappings aren't the only thing that can change the cursor mode.

Right now I would say that it's better to keep this `focus()` call here and move the one in `handlePointerMapping()` into the `.command` branch of the switch.

Reply:
Safe. Done.